f2/src/locales/en.json

```diff
@@ -230,9 +230,6 @@
   "finalFeedback.wasServiceHard.neutral": "Neutral",
   "finalFeedback.wasServiceHard.veryEasy": "Very easy",
   "finalFeedback.wasServiceHard.veryHard": "Very hard",
-  "finalFeedback.wouldYouUseAgain.maybe": "Maybe",
```

Comment:
Why this got removed here. This should be a only version change. Somebody missing precommit hook?

Reply:
The only change I made was the version change. This must be our beloved pre-commit hooks at work again.